Sprint-3/implement/is-valid-triangle.js

```diff
@@ -33,3 +33,17 @@
 // Then it should return true because the input forms a valid triangle.
 
 // This specification outlines the behavior of the isValidTriangle function for different input scenarios, ensuring it properly checks for invalid side lengths and whether they form a valid triangle according to the Triangle Inequality Theorem.
+
+function isValidTriangle(a, b, c) {
+  if (a + b <= c ||  a + c <= b || b + c <= a) {
+    return false;
+  }
+  else if (a <= 0 || b <= 0 || c <= 0) {
```

Comment:
I was trying to fulfill the acceptance criteria 
// scenario: invalid triangle
// Check for Valid Input:
// Given the sides a, b, and c,
// When any of the sides are less than or equal to zero,
// Then it should return false because a triangle cannot have zero or negative side lengths.